Content.Shared/_L5/StaticField/StaticFieldComponent.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Shared.DeviceLinking;
+using Content.Shared.Power;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._L5.StaticField;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class StaticFieldComponent : Component
+{
+    [DataField, AutoNetworkedField]
+    public PowerDeviceVisuals Status = PowerDeviceVisuals.Powered;
```

Comment:
Double-check whether this is actually necessary. This seems redundant to me at a glance.